docs/eind-tot-eind-typesafety.md

```diff
@@ -0,0 +1,36 @@
+# Introductie
```

Comment:
All docs are in English. should we do this in Dutch?

Reply:
Yes we should. We agreed that all developer documentation is in English (and all end-user docs in Dutch).

Reply:
Okay so this is developer documentation so this needs to be english right?

Reply:
That's what I thought, is this not developer documentation, @edgarvonk ?

Reply:
Funny how we switch to english for a issue about english :P 

I think i just blindly looked at docs/inrichting-zaakafhandelcomponent.md and just thought i will write it in dutch. we have some mixed languages at the moment in the docs folder

Reply:
That's what we just dicussed. :-) inrichting-zaakafhandelcomponent.md is documentation for the end-user and therefore must be in Dutch. All developer documentation we write in English. :-)